src/pybind/mgr/dashboard/frontend/src/app/shared/services/doc.service.ts

```diff
@@ -24,8 +24,8 @@ export class DocService {
   }
 
   urlGenerator(section: string, release = 'master'): string {
-    const domain = `http://docs.ceph.com/docs/${release}/`;
-    const domainCeph = `https://ceph.io/`;
+    const docVersion = release === 'master' ? 'latest': release;
+    const domain = `https://docs.ceph.com/en/${docVersion}/`;
```

Comment:
The `domainCeph` variable should be kept.

```suggestion
    const domain = `https://docs.ceph.com/en/${docVersion}/`;
    const domainCeph = `https://ceph.io/`;
```

Reply:
@bk201 fixed and repushed.